python/cuml/test/test_train_test_split.py

```diff
@@ -31,20 +32,17 @@
 
 
 @pytest.mark.parametrize("train_size", [0.2, 0.6, 0.8])
-@pytest.mark.parametrize("shuffle", [True, False])
-def test_split_dataframe(train_size, shuffle):
+def test_split_dataframe(train_size):
     X = cudf.DataFrame({"x": range(100)})
     y = cudf.Series(([0] * (100 // 2)) + ([1] * (100 // 2)))
 
     X_train, X_test, y_train, y_test = train_test_split(
-        X, y, train_size=train_size, shuffle=shuffle
+        X, y, train_size=train_size
     )
     assert len(X_train) == len(y_train) == pytest.approx(train_size * len(X))
     assert (
         len(X_test) == len(y_test) == pytest.approx((1 - train_size) * len(X))
     )
-    assert (all(X_train.index.to_pandas() == y_train.index.to_pandas()))
-    assert (all(X_test.index.to_pandas() == y_test.index.to_pandas()))
```

Comment:
Why are you removing these checks? Why is this test modified in this PR?

Reply:
It was done accidentally. I didn't realise.